File: Elixir/1006.ex
{a, _} = IO.gets("") |> String.trim |> Float.parse
{b, _} = IO.gets("") |> String.trim |> Float.parse
{c, _} = IO.gets("") |> String.trim |> Float.parse
IO.puts("MEDIA = #{:erlang.float_to_binary((a * 2 + b * 3 + c * 5) / 10, [decimals: 1])}")

File: Dart/1006.dart
import 'dart:io';

void main() {
  double a = double.parse(stdin.readLineSync());
  double b = double.parse(stdin.readLineSync());
  double c = double.parse(stdin.readLineSync());
  print("MEDIA = ${((a * 2 + b * 3 + c * 5) / 10).toStringAsFixed(1)}");
}